features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -404,36 +404,95 @@ void UBLOX_AT_CellularStack::clear_socket(CellularSocket *socket)
     }
 }
 
+#ifndef UBX_MDM_SARA_R41XM
```

Comment:
why this one is being skipped? Can you add this detail to the commit message (similar is stated in the description - this is implemented for 2 targets as I understood) ?

Reply:
Yes, these changes are for  `UBLOX_C030_U201` and `UBLOX_C027`. `UBLOX_C030_R41XM` is skipped as it does not support `UPSND `command for getting IP address so instead of using this function it will use parent class function, [get_ip_address](https://github.com/ARMmbed/mbed-os/blob/b77f6b457e91be95f79b180d3fc9d2ca69a20b50/features/cellular/framework/AT/AT_CellularStack.cpp#L61).